### DISCUSSION TITLE
Linking issues to Commits

### DISCUSSION THREAD

**Discussion by TheLayman** [community/community#22871] [New to GitHub]
<p>Other than referencing in comments using <span class="hashtag">#issue</span>, is there any other way to link issues to commits? </p>

**Comment by lee-dohm** [comment, marked as answer]
<p>See the <a href="https://help.github.com/articles/autolinked-references-and-urls/#issues-and-pull-requests" rel="nofollow noopener">help documentation for all the different ways you can link to issues and PRs</a> from various locations.</p>

**Comment by TheLayman** [comment]
<p>In the same way, Can we link an issue from private repo to a commit in public repo&gt;</p>

**Comment by lee-dohm** [comment]
<p>Yes, you can. But the link won’t work for anyone that doesn’t have access to the private repo.</p>
